src/main/java/com/snowflake/kafka/connector/internal/streaming/schemaevolution/iceberg/IcebergColumnTypeMapper.java

```diff
@@ -11,13 +11,61 @@
 
 import com.snowflake.kafka.connector.internal.streaming.schemaevolution.ColumnTypeMapper;
 import net.snowflake.client.jdbc.internal.fasterxml.jackson.databind.JsonNode;
+import org.apache.iceberg.types.Type;
+import org.apache.iceberg.types.Types;
 import org.apache.kafka.connect.data.Date;
 import org.apache.kafka.connect.data.Decimal;
 import org.apache.kafka.connect.data.Schema;
 import org.apache.kafka.connect.data.Time;
 import org.apache.kafka.connect.data.Timestamp;
 
 public class IcebergColumnTypeMapper extends ColumnTypeMapper {
+
+  /**
+   * See <a href="https://docs.snowflake.com/en/user-guide/tables-iceberg-data-types">Data types for
+   * Apache Iceberg™ tables</a>
+   */
+  public static String mapToSnowflakeDataType(Type apacheIcebergType) {
+    switch (apacheIcebergType.typeId()) {
+      case BOOLEAN:
+        return "BOOLEAN";
+      case INTEGER:
+        return "NUMBER(10,0)";
+      case LONG:
+        return "NUMBER(19,0)";
+      case FLOAT:
+      case DOUBLE:
+        return "FLOAT";
+      case DATE:
+        return "DATE";
+      case TIME:
+        return "TIME(6)";
+      case TIMESTAMP:
+        Types.TimestampType timestamp = (Types.TimestampType) apacheIcebergType;
+        return timestamp.shouldAdjustToUTC() ? "TIMESTAMP_LTZ" : "TIMESTAMP";
```

Comment:
Here I didn't type precisiont of a timestamp like ex. TIMESTAMP_LTZ(6). I want server to resolve it. It's confusing because docs mentions that timestamp -> timestam_LTZ/NTZ(**6**) but when I tried it manually it creates a column with ...(9) precision. I must come back to it when It will start to work end 2 end.